src/libraries/System.Net.Http.Json/tests/FunctionalTests/JsonContext/String.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Text.Json.Serialization;
+using System.Text.Json.Serialization.Metadata;
+
+namespace System.Net.Http.Json.Functional.Tests
+{
+    internal partial class JsonContext : JsonSerializerContext
+    {
+        private JsonTypeInfo<string> _String;
+        public JsonTypeInfo<string> String
+        {
+            get
+            {
+                if (_String == null)
```

Comment:
Nit: this class seems to repeat itself a few times in the test code. Could it perhaps be changed into an abstract base class?

Reply:
Per offline chat, these are properties are defined in the same partial class.